sql/src/test/java/io/crate/expression/operator/AndOperatorTest.java

```diff
@@ -40,4 +46,24 @@ public void testEvaluateAndOperator() {
         assertEvaluate("null and false", false);
         assertEvaluate("null and null", null);
     }
+
+    @Test
+    public void test_get_conjunctions_of_predicate_with_2_ands() {
```

Comment:
minor: don't we stick to CamelCase here?

Reply:
Not sure, we somewhere already started using underscores  - I think it would help with readability and also distinguishes tests more from production code.

But I also think you're right in that it should be somewhat consistent. Should I change it here back to camelCase?

Reply:
I've no strong opinion here, mostly I wanted to know if it was done by intent and if so the reasoning.
If we wanna allow/do that, I'd say its fine to do it context wise, renaming all tests now doesn't make much sense to me as the benefit is questionable.

Reply:
I also think the snake case is more readable in test names and I think introducing them gradually would be a good way to go forward